h2o-test-support/src/main/java/water/TestUtil.java

```diff
@@ -1361,9 +1359,9 @@ public static void writeFrameToCSV(String fileNameWithPath, Frame h2oframe, bool
    * @param randomSeed Seed for the random generator (for reproducibility)
    * @return An instance of {@link Vec} with binary weights (either 0.0D or 1.0D, nothing in between).
    */
-  public Vec createRandomBinaryWeightsVec(final long len, final int randomSeed) {
+  public static Vec createRandomBinaryWeightsVec(final long len, final int randomSeed) {
     final Vec weightsVec = Vec.makeZero(len, Vec.T_NUM);
-    final Random random = new Random(randomSeed);
+    final Random random = new RandomUtils.H2ORandomRNG(randomSeed);
```

Comment:
I would rather go through the factory method `getRNG` - I know this is just a test but it is better to follow the good patterns here as well to prevent proliferating this code into the production codebase.

Reply:
Definitely a good idea. Fixed !